### ISSUE TITLE
Last sync timestamp on Android

### ISSUE THREAD

**Issue by onlinemumie** [standardnotes/mobile#326]
In the current version of S. Notes for Android, I have experienced sync problems that require a sign out to fix, but there is no warning that sync has stopped like in the desktop and web apps. A last sync timestamp that is always visible in the Android app would be useful to indicate sync problems.

**Comment by mobitar** [comment]
Good idea 👍 @radko93 we can add this under the user email in the left side menu. A similar date exists in desktop/web in lower right corner.
